### DISCUSSION TITLE
After installation of version 1.9.2, no home page.

### DISCUSSION THREAD

**Discussion by tcreek** [YOURLS/YOURLS#3796] [Q&A] [answered]
I set this up in the admin panel, and in the config.php, put the URL as "https://mysite.xx". Now when I visit the URL, instead of the main site, I am seeing the root directory of the site.

A bug, or an incorrect set up?

**Comment by ozh** [comment, marked as answer]
https://yourls.org/readme.html#FAQ;noindex

**Comment by tcreek** [comment]
Thanks for that. Perhaps the basic/quick install guide should make a note of that to make it a bit more obvious.
